src/doc/News.md

```diff
@@ -0,0 +1,137 @@
+## Dec 16, 2021
+
+-   **API Update**: New search endpoint to search Genome Nexus database.
+    -   For example, search [BRAF p.V600E](https://www.genomenexus.org/search?keyword=BRAF%20p.V600E)
+
+## Nov 04, 2021
+
+-   **Website New Feature**: Add UniProt topology track to variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
```

Comment:
Maybe just remove `\*` here and in the other places? It looks a bit like a footnote indicator
```suggestion
-   **Website New Feature**: Add UniProt topology track to variant page. [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
```

Reply:
@inodb This is added by prettier, so I add `md` file to `.prettierignore`